build.gradle.kts

```diff
@@ -0,0 +1,235 @@
+import org.javamodularity.moduleplugin.extensions.ModularityExtension
+
+plugins {
+    `java-library`
+    `maven-publish`
+    signing
+    jacoco
+    `jvm-test-suite`
+    id("com.netflix.nebula.contacts") version "7.0.1"
+    id("com.netflix.nebula.source-jar") version "20.3.0"
```

Comment:
Mmmhmm. I definitely understand this. Nice!

Reply:
haha - just the netflix cocktail of plugins for creating a maven release that passes sonatype validation